client/components/Navbar.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter, Switch, Route, Link } from "react-router-dom"
+
+import PetTypesList from "./PetTypesList.js"
+import PetShow from "./PetShow.js"
+import AdoptablePetsOfAType from "./AdoptablePetsOfAType.js"
+
+const NavBar = props => {
+  const [petTypes, setPetTypes] = useState([]);
+
+  const fetchPetTypes = async () => {
+    try{
+      const response = await fetch('/api/v1/petTypes');
+      if(!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        throw(new Error(errorMessage));
+      }
+      const petTypesData = await response.json();
+      setPetTypes(petTypesData.petTypes)
+    } catch(error) {
+      console.error(`Error in fetch: ${error.message}`);
+    }
+  }
+
+  useEffect(() => {
+    fetchPetTypes();
+  }, [])
+
+  const linkList = petTypes.map(petType => {
+    let firstLetter = petType.type[0].toUpperCase();
+    let restOfWord = petType.type.slice(1);
+    let typeOfPet = firstLetter + restOfWord;
+    return <Link to={`/pets/${petType.type}`} key={petType.id}>{typeOfPet} </Link>
+  })
+
+  return(
+    <div>
+      <nav>
+        <Link to="/pets" >Home </Link>
+        <Link to="/adoptions/new" >List Pet for Adoption</Link>
+          <div>
+            {linkList}
+          </div>
```

Comment:
Not something I would change now, but since this will be dynamically generated and can get very large as the types of pets stored in the database increases, a nice refactor would be to add certain links within a dropdown button.

Reply:
We were planning on handling this when we get to styling everything! :)